run_full_analysis.py

```
@@ -3269,13 +3291,36 @@ def generate_report_index(
+    generated_sector_levels: Dict[str, Set[int]] = defaultdict(set)
+    for credential in credentials:
+        sector = str(credential.get("sector", "")).strip()
+        eqf_level = credential.get("eqf_level")
+        if sector and isinstance(eqf_level, int):
+            generated_sector_levels[sector].add(eqf_level)
+    covered_sectors = len(generated_sector_levels)
+    covered_levels = sorted({level for levels in generated_sector_levels.values() for level in levels})
+    review_required_sectors = sorted(
+        {
+            str(item.get("sector", "")).strip()
+            for item in review_required
```

Ruff (0.16.1)

[error] Undefined name `review_required`

(F821)